keyboards/wuque/nemui65/keymaps/via/keymap.c

```diff
@@ -32,3 +32,69 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
         _______, _______, _______,                            _______,                   _______, _______,          _______, _______, _______
     )
 };
+
+#ifdef RGB_MATRIX_ENABLE
+
+enum { other = 0, mac = 1, win = 2 } system_type;
+enum { none = 0, mac_led = 1, win_led = 2 } LED_FLAG = none;
+
+void keyboard_post_init_user() {
+    if (dynamic_keymap_get_keycode(0, 4, 1) == KC_LGUI && dynamic_keymap_get_keycode(0, 4, 2) == KC_LALT && dynamic_keymap_get_keycode(0, 4, 10) == KC_RALT) {
+        system_type = win;
+    } else if (dynamic_keymap_get_keycode(0, 4, 1) == KC_LALT && dynamic_keymap_get_keycode(0, 4, 2) == KC_LGUI && dynamic_keymap_get_keycode(0, 4, 10) == KC_RGUI) {
+        system_type = mac;
+    } else {
+        system_type = other;
+    }
+}
```

Comment:
Would the [OS detection feature](https://docs.qmk.fm/#/feature_os_detection) be a better fit? Checking if a keycode exists in certain location doesn't seem like the most reliable method.